addons/elasticsearch-cluster/values.schema.json

```diff
@@ -0,0 +1,168 @@
+{
+  "$schema": "http://json-schema.org/schema#",
+  "type": "object",
+  "properties": {
+    "replicaCount": {
+      "title": "ReplicaCount",
+      "description": "The number of replicas.",
+      "type": "integer",
+      "default": 1
+    },
+    "mode": {
+      "title": "Mode",
+      "description": "Cluster topology mode.",
+      "type": "string",
+      "default": "multi-node",
+      "enum": [
+        "single-node",
+        "multi-node"
+      ]
+    },
+    "terminationPolicy": {
+      "title": "TerminationPolicy",
+      "description": "The policy of termination.",
+      "type": "string",
+      "default": "Delete",
+      "enum": [
+        "Delete",
+        "DoNotTerminate",
+        "Halt",
+        "WipeOut"
+      ]
+    },
+    "imagePullSecrets": {
+      "title": "ImagePullSecrets",
+      "type": "array",
+      "items": {
+        "type": "string"
+      }
+    },
+    "nameOverride": {
+      "title": "NameOverride",
+      "description": "The override name.",
+      "type": "string",
+      "default": ""
+    },
+    "fullnameOverride": {
+      "title": "FullnameOverride",
+      "description": "The override fullname.",
+      "type": "string",
+      "default": ""
+    },
+    "resources": {
+      "title": "resources",
+      "description": "If you do want to specify resources, uncomment the following lines after 'resources:' in values.yaml",
+      "type": "object"
+    },
+    "nodeSelector": {
+      "title": "NodeSelector",
+      "type": "object"
+    },
+    "tolerations": {
+      "title": "Tolerations",
+      "type": "array",
+      "items": {}
+    },
+    "affinity": {
+      "title": "Affinity",
+      "type": "object"
+    },
+    "topologyKeys": {
+      "title": "TopologyKeys",
+      "type": "array",
+      "items": {
+        "type": "string",
+        "default": "kubernetes.io/hostname"
+      }
+    },
+    "monitor": {
+      "title": "Monitor",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "default": false
+        }
+      }
+    },
+    "persistence": {
+      "title": "Persistence",
+      "description": "Persistence config.",
+      "type": "object",
+      "properties": {
+        "enabled": {
+          "type": "boolean",
+          "default": true
+        },
+        "data": {
+          "type": "object",
+          "properties": {
+            "storageClassName": {
+              "type": ["string", "null"],
+              "default": null
+            },
+            "size": {
```

Comment:
`size` cannot be an arbitary string. You may refer to 
https://github.com/apecloud/kubeblocks-addons/blob/676f181686f5018a19608c51f887b554cbbc9f00/addons/apecloud-mysql-cluster/values.schema.json#L29-L30

Reply:
Thanks for reviewing! I've added limitations for both replicas&resources&persistence